File: Serene3/Serene3.Web/Modules/ServerTypes/com/cartColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { cartRow } from "./cartRow";

export interface cartColumns {
    Id: Column<cartRow>;
    ProductidName: Column<cartRow>;
    Userid: Column<cartRow>;
    Photo: Column<cartRow>;
    Price: Column<cartRow>;
    Quntity: Column<cartRow>;
}

export class cartColumns extends ColumnsBase<cartRow> {
    static readonly columnsKey = 'com.cart';
    static readonly Fields = fieldsProxy<cartColumns>();
}

File: Serene3/Serene3.Web/Modules/com/cart/cartColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.cart")]
[BasedOnRow(typeof(cartRow), CheckNames = true)]
public class cartColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public string ProductidName { get; set; }
    [EditLink]
    public string Userid { get; set; }
    public string Photo { get; set; }
    public decimal Price { get; set; }
    public int Quntity { get; set; }
}